src/app/zap-templates/templates/app/CHIPClusters-src.zapt

```diff
@@ -13,7 +13,7 @@ namespace Controller {
 // TODO(#4503): length should be passed to commands when byte string is in argument list.
 // TODO(#4503): Commands should take group id as an argument.
 
-{{#chip_server_clusters}}
+{{#chip_client_clusters}}
```

Comment:
Why the changes from server to client here and further down?

Reply:
Because now the files will be generated on a per example basis, and the commands are generated based on the list of `client` clusters selected by the example. For example, `chip-tool` has many `client` clusters, but no `server`. Same things for `iOS ChipTool` and the Python device controller.

The fact that we still use `chip_server_clusters` here is historical. At the origin this file was generated from the `all-clusters-app` configuration.

Reply:
Gotcha.

Reply:
For controller, I guess we still need `chip_client_clusters`, since we are sending commands to the server and we need to encode the commands supported by **server**